src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -157,123 +161,130 @@ docker run -it --rm -v /var/run/docker.sock:/var/run/docker.sock
+*Example*
 
-----
-# Get the new version of Che
-docker pull eclipse/che-cli:6.0.0
+If you have 6.0.0 installed and want to upgrade to 6.0.1:
 
-# You now have two eclipse/che-cli images (one for each version)
-# Perform an upgrade - use the new image to upgrade old installation
-docker run <volume-mounts> eclipse/che-cli:6.0.1 upgrade
-----
+. Get the new version of Che: `docker pull eclipse/che-cli:6.0.0`.
++
+You now have two eclipse/che-cli images (one for each version).
++
+. Use the new image to upgrade the old installation: `docker run <volume-mounts> eclipse/che-cli:6.0.1 upgrade`.
++
+The upgrade command has numerous checks to prevent you from upgrading Che if the new image and the old one are not compatible. For the upgrade procedure to advance, the CLI image must be newer than the one in the `/instance/che.ver` file.
 
-The upgrade command has numerous checks to prevent you from upgrading Che if the new image and the old version are not compatible. In order for the upgrade procedure to advance, the CLI image must be newer that the version in `/instance/che.ver`.
+To upgrade Che, take the following steps:
 
-The upgrade process:
+. Perform a version compatibility check.
+. Download the new Docker image needed to run the new version of Che.
+. If it is running, stop Che.
+. Trigger a maintenance window.
+. Take a backup of your installation.
+. Initialize the new version.
+. Start Che.
 
-1.  Performs a version compatibility check
-2.  Downloads new Docker images that are needed to run the new version of Che
-3.  Stops Che if it is currently running
-4.  Triggers a maintenance window
-5.  Backs up your installation
-6.  Initializes the new version
-7.  Starts Che
-8.  Important! If `CHE_PREDEFINED_STACKS_RELOAD__ON__START` is set to false, stacks packaged into new binaries will not be saved into a database.
+[IMPORTANT]
+====
+If the `CHE_PREDEFINED_STACKS_RELOAD__ON__START` variable is set to false, stacks packaged into the new binaries will not be saved in the database.
+====
 
 [id="backup"]
-== Backup
+== Taking a backup
 
-You can run `che backup` to create a copy of the relevant configuration information, user data, projects, and workspaces. We do not save workspace snapshots as part of a routine backup exercise. You can run `che restore` to recover Che from a particular backup snapshot. The backup is saved as a TAR file that you can keep in your records. You can then use `che restore` to recover your user data and configuration.
+* To create a copy of the relevant configuration information, user data, projects, and workspaces, run the `che backup` command. As part of a routine backup exercise, Che does not save workspace snapshots. 
+* To recover Che from a particular backup snapshot, run the `che restore` command. The backup is saved as a TAR file that you can keep in your records. You can then use the `che restore` command to recover your user data and configuration.
```

Comment:
These commands need examples. That's out of scope for this review, but please file an issue to fix it in the future.

Reply:
Created at https://github.com/eclipse/che-docs/issues/521.